tests/behat/features/web/mediapackage.feature

```diff
@@ -56,3 +46,21 @@ Feature:
     And I should see media file with id 7 in category "Luna & Cat Theme Special"
     And I should see 1 media file in category "Luna & Cat Theme Special"
     And I should see 1 media file in category "Bla"
+
+  Scenario: Selecting and deselecting media files for downloading
+    Given I am on "/app/media-library/looks"
+    And I click "#mediafile-1"
+    And I click "#mediafile-5"
+    Then I should see "Download these files"
+    And I click "#mediafile-1"
+    And I click "#mediafile-5"
+    Then I should not see "Download these files"
+
+  Scenario: Downloading mutiple selected files
+    Given I am on "/app/media-library/looks"
+    And I click "#mediafile-1"
+    And I click "#mediafile-5"
+    And I click "#start-downloads"
+    And I wait 5000 milliseconds
```

Comment:
is it possible to wait for a response or something?

Reply:
Now implementend waiting for the downloads to be finished.